README.md

```diff
@@ -149,7 +149,7 @@ A curated list of awesome [remote working](https://en.wikipedia.org/wiki/Telecom
   1. [Dribbble Jobs](https://dribbble.com/jobs?location=Anywhere)
   1. [Drupal Jobs](https://jobs.drupal.org/home/type/telecommute-remote-3588)
   1. [Find Bacon](https://findbacon.com) - Design and Dev jobs
-  1. [Flexjobs](https://www.flexjobs.com/) – Telecommuting Jobs & Professional Part-Time Jobs.
+  1. [Flexjobs](https://www.flexjobs.com/) – Telecommuting Jobs & Professional Part-Time Jobs. Requires paid membership to see all results.

```

Comment:
why file is required

Reply:
Do you mean why this change should exist? I think when you have a list of job boards there should be some transparency that a board will require $$$ to properly use.

Reply:
Nice👍👏